src/map/enmity_container.cpp

```diff
@@ -329,18 +328,12 @@ void CEnmityContainer::SetVE(CBattleEntity* PEntity, uint16 amount)
 *                                                                       *
 ************************************************************************/
 
-void CEnmityContainer::UpdateEnmityFromDamage(CBattleEntity* PEntity, uint16 Damage)
+void CEnmityContainer::UpdateEnmityFromDamage(CBattleEntity* PEntity, int32 Damage)
 {
     Damage = (Damage < 1 ? 1 : Damage);
 
-    uint16 mod = battleutils::GetEnmityModDamage(PEntity->GetMLevel()); //default fallback
-
-    if (m_EnmityHolder != nullptr) {//use the correct mod value
-        mod = battleutils::GetEnmityModDamage(m_EnmityHolder->GetMLevel());
-    }
-
-    auto CE = (int16)((80.0f / mod) * Damage);
-    auto VE = (int16)((240.0f / mod) * Damage);
+    int32 CE = 80 / battleutils::GetEnmityModDamage(m_EnmityHolder->GetMLevel()) * Damage;
```

Comment:
the truncation in these changes are different, is that intended? previously, truncation only happened at the end - now, truncation is happening before being multiplied by damage, which means that CE can only be a whole multiple of damage, and then VE can only be a whole multiple of (damage * 3)

Reply:
you right. I fix.

Reply:
I'll fix other spots, too.